tests/common/scripts/bookinfo.txt

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
This script should use a `snips.sh` file generated from `/docs/examples/bookinfo/index.md`